static/frontend/Caddyfile

```diff
@@ -0,0 +1,30 @@
+{
+	admin off
+	http_port 8080
+	https_port 8443
+
+	# Do not add the webserver's local certificate authority to the container's trust store
+	skip_install_trust
+}
+
+localhost {
```

Comment:
Todo: to make this configuration suitable for use with arbitarily-chosen domain names, this server address should be updated to expand from an environment variable.

That environment variable should have a default value of `localhost` specified in the Containerfile.

Reply:
Very good point.